api/config/sample_data/config_request.proto

```diff
@@ -0,0 +1,41 @@
+syntax = "proto3";
+
+package chef.automate.domain.sample_data;
+option go_package = "github.com/chef/automate/api/config/sample_data";
+
+import "config/shared/global.proto";
+import "config/shared/tls.proto";
+import "automate-grpc/protoc-gen-a2-config/api/a2conf/annotations.proto";
+import "google/protobuf/wrappers.proto";
+
+message ConfigRequest {
+	option (chef.automate.api.service_config) = {name: "sample-data-service"};
+
+	V1 v1 = 1;
+
+	message V1 {
+		System sys = 1;
+		Service svc = 2;
+
+		message System {
+			chef.automate.infra.config.Mlsa mlsa = 1;
+			Service service = 2;
+			chef.automate.infra.config.TLSCredentials tls = 3;
+			Log log = 4;
+
+			message Service {
+				google.protobuf.StringValue host = 1 [deprecated=true]; // The listen host is no longer setable(localhost only)
```

Comment:
We can avoid adding a deprecated option at the start of this service since no users have used this option yet.